### PR TITLE
ARSN-195: fix arsenal bugs and missing default in require

### PR DESCRIPTION
**Pull request by ghost**
This PR fixes some misses and bugs from https://github.com/scality/Arsenal/pull/1844.

Related ANDON: https://scality.atlassian.net/browse/AN-136

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -63,8 +63,8 @@ export const s3routes = {
 
 export const s3middleware = {
     userMetadata: require('./lib/s3middleware/userMetadata'),
-    convertToXml: require('./lib/s3middleware/convertToXml'),
-    escapeForXml: require('./lib/s3middleware/escapeForXml'),
+    convertToXml: require('./lib/s3middleware/convertToXml').default,
```
Comment (jbertran) [inline index.ts]: The real fix for these is switching to `import/export` in `index.ts`, similar to how `policies` are exported.

**Review by jbertran — CHANGES_REQUESTED** [review]

File: lib/s3middleware/userMetadata.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function getMetaHeaders(headers: http.IncomingHttpHeaders) {
         if (!v) return length;
         return length + k.length + v.toString().length;
     }, 0);
-    if (totalLength < constants.maximumMetaHeadersSize) {
+    if (totalLength <= constants.maximumMetaHeadersSize) {
```
Comment (jbertran) [inline lib/s3middleware/userMetadata.ts:17]: Ideally there's a unit test to be done here, so this doesn't happen in the future

Comment (ghost) [reply lib/s3middleware/userMetadata.ts:17]: Done in https://github.com/scality/Arsenal/pull/1882/commits/45945789193981d8e83a83aa0773b1f1ea5c5af8

File: index.ts (hunk 1 above)
Comment (ghost) [reply index.ts]: Done in https://github.com/scality/Arsenal/pull/1882/commits/9e0cee849cba2a7607d7ca41ecc0dfaae51b7d38

**Review by jbertran — APPROVED** [review]

**Review by XinLi0207 — APPROVED** [review]

**Pull request merged**